packages/studio-ui/src/web/components/Layout/TopNav/cloud/types.ts

```diff
@@ -0,0 +1,13 @@
+import { NLU } from 'botpress/sdk'
+
+/**
+ * CDM Workspace
+ */
+export interface Workspace {
```

Comment:
**suggestion :**
```ts
export interface BPCloudWorkspace {
  id: string
  name: string
}
```

Reply:
It's already in the `cloud/` directory, thus we should assume it's cloud-related

Reply:
I could rename it to `CDMWorkspace` though

Reply:
Fixed in fb992a7